### PR TITLE
File Copy Change Time

### PR DESCRIPTION
**Pull request by ibrahimrabab**
Continued from https://github.com/Azure/azure-sdk-for-java/pull/28718

### CODE REVIEW

**Review by seanmcc-msft — APPROVED** [review]

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy [hunk 1]
```diff
@@ -1052,6 +1053,274 @@ class FileAPITests extends APISpec {
...
+        SyncPoller<ShareFileCopyInfo, Void> poller = primaryFileClient.beginCopy(sourceURL, options)
+
+        def pollResponse = poller.poll()
+
+        then:
+        pollResponse.getValue().getCopyId() != null
+        pollResponse.getStatus() == LongRunningOperationStatus.SUCCESSFULLY_COMPLETED
+    }
+
+    def "Start copy with options file permission"() {
```
Comment (rickle-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy:1110]: Are most of these tests duplicates of existing tests but using the options?

Comment (ibrahimrabab) [reply sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy:1110]: Yup basically, I'm just asserting the results of passing the necessary fields in options. Is this redundant?

Comment (rickle-msft) [reply sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy:1110]: Personally I think it is, yes, and would only add tests for the new feature being added

**Review by rickle-msft — APPROVED** [review]

**Pull request merged**